.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -121,34 +106,18 @@ body:
       label: "Nightwatch.js Version"
       description: "The version of Nightwatch you use."
       placeholder: "2.3.0"
-    validations:
-      required: true
```

Comment:
I think we should keep the Nightwatch version as required and just add the command which the user can use to get the version of Nightwatch.

Also, we should add a command in the description of all below version questions which the user can use to get those version numbers.

Reply:
we definitely want a version number as mandatory

Reply:
updated